tests/integration/commands/multicall_command_test.py

```diff
@@ -0,0 +1,86 @@
+import json
+from pathlib import Path
+
+import pytest
+from starknet_py.net.models import StarknetChainId
+from pytest import CaptureFixture
+
+from protostar.starknet_gateway.multicall import (
+    prepare_multicall_file_example,
+    Identifier,
+)
+from tests.integration.conftest import CreateProtostarProjectFixture
+from tests.integration.protostar_fixture import ProtostarFixture
+from tests.conftest import DevnetFixture, SetPrivateKeyEnvVarFixture
+
+
+@pytest.fixture(name="protostar")
+def protostar_fixture(create_protostar_project: CreateProtostarProjectFixture):
+    with create_protostar_project() as protostar:
+        protostar.build_sync()
+        yield protostar
+
+
+@pytest.fixture(name="multicall_file_path")
+async def multicall_file_path_fixture(
+    protostar: ProtostarFixture,
+    devnet: DevnetFixture,
+    set_private_key_env_var: SetPrivateKeyEnvVarFixture,
+    capsys: CaptureFixture[str],
+    tmp_path: Path,
+) -> Path:
+    account = devnet.get_predeployed_accounts()[0]
+    with set_private_key_env_var(account.private_key):
+        declare_result = await protostar.declare(
+            contract=Path() / "build" / "main.json",
+            account_address=account.address,
+            chain_id=StarknetChainId.TESTNET,
+            wait_for_acceptance=True,
+            gateway_url=devnet.get_gateway_url(),
+            max_fee="auto",
+        )
+    capsys.readouterr()
+    multicall_doc_path = tmp_path / "multicall.toml"
+    file_content = prepare_multicall_file_example(class_hash=declare_result.class_hash)
```

Comment:
We should test at least a few examples of structure here, to have more comprehensive tests